account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -157,8 +157,21 @@ def create_company(self):
         self.new_company_id = self.env["res.company"].create(
             {"name": self.name, "user_ids": [(6, 0, self.user_ids.ids)]}
         )
-        self.new_company_id.chart_template_id = self.chart_template_id
-        self.new_company_id.chart_template_id.try_loading_for_current_company()
+        allowed_company_ids = (
+            self.env.context.get("allowed_company_ids", []) + self.new_company_id.ids
+        )
+        new_company_id = self.new_company_id.with_context(
+            allowed_company_ids=allowed_company_ids
+        )
+        self.env.user.write(
+            {
+                "company_ids": [(4, self.new_company_id.id)],
+                "company_id": self.new_company_id.id,
```

Comment:
This change is too invasive. I don't get why this is needed and in other versions it's not. Maybe you can check the current 14.0 migration PR to see if there's the same problem.

Reply:
True, this is not really necessary.
In the PR of 14.0 this problem does not exist because as this in v13 it does not work (it creates the company but not the data of the accounting template).